doc/manual/src/glossary.md

```diff
@@ -168,9 +181,10 @@
 
   A high-level description of software packages and compositions
   thereof. Deploying software using Nix entails writing Nix
-  expressions for your packages. Nix expressions are translated to
-  derivations that are stored in the Nix store. These derivations can
-  then be built.
+  expressions for your packages. Nix expressions specify [derviations],
+  which are [instantiated] and stored in the Nix store.
+  These derivations can then be [realised][realise] (built), and produce
```

Comment:
Multiline suggestion again: "... , to produce the derivation [outputs][output]".